webkit2/cl-webkit2.asd

```diff
@@ -20,7 +20,7 @@
   :description "An FFI binding to WebKit2GTK+"
   :author "Joachim Fasting <joachifm@fastmail.fm>"
   :licence "MIT"
-  :version "0.0"
+  :version "3.0"
```

Comment:
What about using semver from now on and thus 3.0.0 instead?

Reply:
Done in 4a72f251c17566a82d494f11af1f4cea79c58c3f.